cg/store/status_collaboration_filters.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+from typing import List, Optional, Callable
+
+from sqlalchemy.orm import Query
+
+from cg.store.models import Collaboration
+
+
+def get_collaboration_by_internal_id(collaborations: Query, internal_id: str, **kwargs) -> Query:
+    """Return collaboration by internal_id."""
+    return collaborations.filter(Collaboration.internal_id == internal_id)
+
+
+class CollaborationFilters(Enum):
+    """Define Collaboration filter functions."""
+
+    get_collaboration_by_internal_id: Callable = get_collaboration_by_internal_id
```

Comment:
Enums are usually named in all [uppercase](https://mail.python.org/pipermail/python-ideas/2016-September/042340.html). It is applied inconsistently, but maybe we could start following that convention. Also, I think it is more readable if we name them like ```FILTER_BY_ID``` instead of ```get_collaboration_by_internal_id```. It will always be used with the name of the enum first ```CollaborationFilters.get_collaboration_by_internal_id```, so "collaboration" seems redundant.

Reply:
So we're not talking about the class name here? Something like this?
```suggestion
    FILTER_BY_ID: Callable = get_collaboration_by_internal_id
```

Reply:
Exactly 💯  I meant the fields of the enum class.